src/components/atoms/Input/Input.model.ts

```diff
@@ -2,10 +2,16 @@ type InputType = 'text' | 'password' | 'email' | 'date' | 'number';
 
 export interface IProps {
```

Comment:
Each props related to html element should be extended by `HTMLElement`. This is specified in [FRDQ-155](https://app.vivifyscrum.com/boards/97818/FRDQ-155). If we do this, then we dont need here `type`, `placeholder`, `min`, `max`, `value`, `pattern`, `className`, `ref`, `inputRef`, `defaultValue`, `onChange`, `onFocus`